gatsby/static/css/webflow-overrides.css

```diff
@@ -33,6 +33,29 @@ h1, h2, h3, h4, h5, h6 {
   font-weight: bold;
 }
 
+p:not(:last-child),
+blockquote:not(:last-child),
+figure:not(:last-child),
+ul:not(:last-child),
+ol:not(:last-child) {
+  margin-bottom: 1em;
+}
```

Comment:
Only put margin in between elements.

Lots of different ways to tackle this sort of thing like the [lobotomized owl selector](https://alistapart.com/article/axiomatic-css-and-lobotomized-owls/).